Numerous detective related changes.

pireamaineach opened the pull request:
Cleans up the kit of detective plasmamen, the addition of the loose tie allowing them to easily remove the tie, preventing clipping issues with their envirosuit. The white helmet provides them with more versatility to wear clothing such as the noir coat, and the bowman headset was replaced to prevent clipping issues with the helmet.
(After removing the tie) New:
![New](https://user-images.githubusercontent.com/46943530/54885569-fddecd00-4e53-11e9-886f-ea75ae812a5b.png)
Old:
![Old](https://user-images.githubusercontent.com/46943530/54885578-0cc57f80-4e54-11e9-8c1b-116ac0904307.png)


New Noir Suit:
![NoirSuit](https://user-images.githubusercontent.com/46943530/54958171-41f0d100-4f2b-11e9-80e1-59d989c1e416.png)
Edited Noir Trenchcoat:
![NoirTrenchcoat](https://user-images.githubusercontent.com/46943530/54958446-14f0ee00-4f2c-11e9-8a81-93eb54d1375f.png)


Revolver Sprites:
![Revolvers](https://user-images.githubusercontent.com/46943530/54958872-b3ca1a00-4f2d-11e9-880c-9b6f869ee48c.png)


:cl:
Adds the loose tie.
Adds the noir suit coat.
Noir suit coat now spawns in the detective's locker.
The detective spawns with a loose tie.
Removes the tie from the trenchcoat's sprites, the loose tie mirroring it exactly.
Plasmaman detectives spawn with a white helmet.
Plasmaman detectives now spawn with a normal security headset.
Resprites/Recolours the noir trench coat, making it darker to match the fedora.
Resprites the colt detective special to actually look like a colt detective special.
Adds two new reskin options for the detective special inspired by historical variants of the detective special, the police positive and fitz special, as well as blued and stainless steel variants.
Renames the "Mars .38 Special" to Colt Detective Special because there's no real reason why it shouldn't be the actual name.
/:cl:

pireamaineach committed:
Numerous detective and plasmaman related changes.

pireamaineach committed:
help

I never actually intended to remove this I assume I shouldn't.

pireamaineach committed:
help v2.0

i didnt mean to please don't beat me again dad pleASE

pireamaineach committed:
Merge branch 'detectiveboy' of https://github.com/pireamaineach/tgstation into detectiveboy

as334 commented:
Nice job. This looks a lot nicer.

pireamaineach committed:
Revolver Changes

Makes the detective special actually look like a detective special, adds two historically inspired variants of the detective special, the fitz and the police positive special.
The detective special is now actually a called a detective special.
Also sorts the list of reskin options for the detective special by my own tier list because fuck you.

pireamaineach committed:
Unironically only edits one pixel.

I want die.

pireamaineach committed:
Noir suit resprites.

pireamaineach committed:
Makes the noir suit coat's length consistent.

I swear I'm not disabled.

pireamaineach commented:
Now it's actually ready; don't quote me. I should probably not commit every time I make a tiny change but fuck it.

subject217 commented:
pretty bikesheddy tbh, are you making these for your static character

pireamaineach commented:
I main atmos tech and the noir suit changes were at the request of a detective main.

subject217 commented:
🤷‍♀️ 

i am drunk so i am probably crazy shouldtn comment on prs tbh

pireamaineach commented:
I like that with this comment he kept the downvote.

anconfuzedrock commented:
brb downvoting

pireamaineach commented:
my dad works at github ill ban u kid

tralezab commented:
You did the changelog wrong

Also please dont rename the detective gun, the last one was fine. Otherwise I like it I guess

Arianya commented:
Have you checked whether the Colt Detective Special's name/brand is copyrighted? It might seem like a stupid question but I recall this coming up in the past elsewhere in relation to well known guns and the manufacturer's being copyright happy.

Also I'm not crazy about the new noir suit. is there any reason to replace the old noir suit and not make this a new outfit? (and probably give the Detective a clothing vendor, it may be overdue).

pireamaineach commented:
> Have you checked whether the Colt Detective Special's name/brand is copyrighted? It might seem like a stupid question but I recall this coming up in the past elsewhere in relation to well known guns and the manufacturers being copyright happy.
> 
> Also I'm not crazy about the new noir suit. is there any reason to replace the old noir suit and not make this a new outfit? (and probably give the Detective a clothing vendor, it may be overdue).

Colt isn't copyright happy in the slightest and we've several times used the name of their other models, notably the m1911.
Clothing vendor seems unnecessary and the noir suit was by request, I don't know anyone who's particularly favourable to the current noir suit anyway.

pireamaineach commented:
> You did the changelog wrong
> 
> Also please dont rename the detective gun, the last one was fine. Otherwise I like it I guess

The new name is just as fine, it's not a significant change.
It's my changelog I'll do what I want.

Arianya commented:
I like the trenchcoat, and one person requesting a change isn't really justification for removal of an outfit and replacing it with another.

Not in a position to check it personally, but by the way you're talking about it, the waistcoat is also baked into into the new outfit instead of being optional? Thats a big :-1: from me considering we already have the optional waistcoat.

Also changing balance (bowman vs normal headset) purely for aesthetic is fairly :confused:. I doubt anyone will complain given the limited cross section of detective players and plasmaman players but worth noting.

pireamaineach commented:
> I like the trenchcoat, and one person requesting a change isn't really justification for removal of an outfit and replacing it with another.
> 
> Not in a position to check it personally, but by the way you're talking about it, the waistcoat is also baked into into the new outfit instead of being optional? Thats a big  from me considering we already have the optional waistcoat.
> 
> Also changing balance (bowman vs normal headset) purely for aesthetic is fairly . I doubt anyone will complain given the limited cross section of detective players and plasmaman players but worth noting.

The waistcoat is still optional and the plasmaman envirohelmet protects the ears.

ExcessiveUseOfCobblestone commented:
I actually prefer the old noir jacket :(

pireamaineach commented:
> I actually prefer the old noir jacket :(

I'll see what I can do.

PKPenguin321 commented:
>revolver is one of the few items with a versatile custom sprites system so we can add new ones without replacing the old ones
>uhhhhh lets replace the old ones anyways lol

Dislike the new gun sprites, the contrast is way too low and it makes all the detail muddy-looking and mesh together, but you could still add them as alt skins
Also WHITE HELMET WTF

pireamaineach commented:
> > revolver is one of the few items with a versatile custom sprites system so we can add new ones without replacing the old ones
> > uhhhhh lets replace the old ones anyways lol
> 
> Dislike the new gun sprites, the contrast is way too low and it makes all the detail muddy-looking and mesh together, but you could still add them as alt skins
> Also WHITE HELMET WTF

It's because you're looking at it on a white background.
Also I'm supposed to add an alternate sprite of the colt detective special that actually looks like a colt detective special, instead of just respriting it?
Tell you what though, I should add an alternate sprite that's silver as it orignally was.

ExcessiveUseOfCobblestone commented:
The gun should be able to be renamed so it doesn't _really_ matter how 1:1 it is with the initial name of the gun.

pireamaineach commented:
Yeah, I just don't see why it shouldn't be 1:1. Again, it's a minor change.

ExcessiveUseOfCobblestone commented:
Because I want it to be the best bikeshed this game has to offer.

pireamaineach commented:
Alright boys I've decided that to appease PKpenguin's being autistic I shall instead of just flat-out improving the visuals of the detective's gun I will leave in the out-dated sprites and add the new ones as alternate skins even though it's incredibly redundant.

pireamaineach commented:
God bless.

pireamaineach commented:
Unironically if you have a problem with the P.R. take it up with me and I'll see about finding a solution.

ExcessiveUseOfCobblestone commented:
(* ´>艸<)゛

PKPenguin321 commented:
Thanks dude, careful with saying a****** here though or githubs robots will autoban you

pireamaineach commented:
> Thanks dude, careful with saying a****** here though or githubs robots will autoban you

No problem I'm definitely doing this 100%.

pireamaineach commented:
God damn it moo stop being silly.

pireamaineach committed:
More revolvers options, noir suit as a separate item.

pireamaineach commented:
Does this please the masses?

pireamaineach commented:
I swear to fucking god if I put on this ready tag and you all start complaining.

pireamaineach committed:
Fixes inconsistent back on the enviroslacks.

Arianya commented:
Good to have the trenchcoat. Flared bottom looks a bit weird on github screenshot but might work fine in game?

pireamaineach commented:
"I like the trenchcoat," says Arianya.
"The trenchcoat looks a bit weird," says Arianya.

pireamaineach committed:
Updates the shotgun sprites to the new ones.

I have no idea what I'm fucking doing my lack of knowledge on Git is really biting me in the ass someone please send help.

pireamaineach committed:
Please work.

pireamaineach commented:
What the fuck why?

pireamaineach committed:
whoops

pireamaineach committed:
whoops

pireamaineach committed:
Changes the order of the sprites in the .dmi.

I'm going to be real I just wanted to check on something. Hopefully the last commit.

pireamaineach commented:
wHy

ExcessiveUseOfCobblestone commented:
press f

pireamaineach commented:
This wouldn't even matter if it was merged it's nearly exactly the same what the fuck. Am I just going to have the merge conflict tag and a red x because of github hack chicanery?

pireamaineach committed:
FuCK

shit

pireamaineach commented:
Mother fucker--

pireamaineach committed:
Please.

h

pireamaineach committed:
h

h

pireamaineach committed:
I'm done.

pireamaineach commented:
This just isn't going to work, fuck it. Barely even a conflict, the new file even has the changes made.

pireamaineach commented:
Even the sprite difference bot gave up on this P.R.

pireamaineach committed:
Once more.

pireamaineach commented:
Icon difference bot is back, everything seems fine enough besides me having wasted a lot of time.

ATH1909 commented:
Maybe the detective should get its own drop pod beacon for detective outfits, like what the chaplain gets for their holy armor?

Maybe you could make the kind of armor that you spawn with as a detective a part of your preferences, sort of like how sec officers can choose their starting department in their preferences?

pireamaineach commented:
Seems very, very unnecessary considering the locker works just fine.

pireamaineach committed:
At this point I'm not even going to try and name these commits.

pireamaineach committed:
Update projectile.dmi

pireamaineach commented:
What the fuck. It literally just doesn't let me update the file at all.

pireamaineach committed:
Update projectile.dmi

pireamaineach committed:
Merge remote-tracking branch 'upstream/master' into detectiveboy

pireamaineach committed:
Minimal buff to the .38, bringing it to par with 10mm, though not the stetchkin.

Makes it mildly more effective at dropping armoured targets, but effectively the same from 0 damage to an unarmoured target.

pireamaineach commented:
english is my second language

PKPenguin321 commented on code/modules/projectiles/projectile/bullets/revolver.dm:
```diff
@@ -14,7 +14,7 @@
 /obj/item/projectile/bullet/c38
 	name = ".38 bullet"
-	damage = 25
+	damage = 30
```
seems kind of like a stealth change

pireamaineach replied:
Real' stealthy the way I asked people ahead of time and specifically noted it in the commit.

PKPenguin321 replied:
not in PR body or changelog or anywhere but the commit and in a PR that's primarily for cosmetic changes

pireamaineach replied:
Sorry that I didn't update the changelog after all this shit. I just want this to be over with, the merge conflict was a headache.

ExcessiveUseOfCobblestone replied:
revert this and put it in a separate PR then i'll merge this PR

```suggestion
	damage = 25
```

pireamaineach commented:
Real' stealthy the way I asked people ahead of time and specifically noted it in the commit.

ExcessiveUseOfCobblestone committed:
Update code/modules/projectiles/projectile/bullets/revolver.dm

Okay I guess.

Co-Authored-By: pireamaineach <46943530+pireamaineach@users.noreply.github.com>
